extra/alerts/alert_snmp.sh.sample

```diff
@@ -92,7 +92,18 @@ trap_snmp_persistent_dir_default="/var/lib/pacemaker/snmp"
 : ${trap_snmp_persistent_dir=${trap_snmp_persistent_dir_default}}
 
 if [ "${trap_add_hires_timestamp_oid}" = "true" ]; then
-    hires_timestamp="HOST-RESOURCES-MIB::hrSystemDate s ${CRM_alert_timestamp}"
+# 
+# Then disable the set of hires_timestamp using the next CRM_alert_timestamp_epoch.
+# According to the format of DateAndTime of SNMP, hires_timestamp can be changed by script.
+# The current format is "%Y-%m-%d,%H:%M:%S.%01N".
+#
+    trap_fmt_timestamp=`date --date="@${CRM_alert_timestamp_epoch}" +"%Y-%m-%d,%H:%M:%S."`$(echo ${CRM_alert_timestamp_ns} | cut -b 1)
+    hires_timestamp="HOST-RESOURCES-MIB::hrSystemDate s ${trap_fmt_timestamp}"
+#
+# If you want an old style format specification, please remove the following comment.
+#
+#    hires_timestamp="HOST-RESOURCES-MIB::hrSystemDate s ${CRM_alert_timestamp}"
+# 
```

Comment:
I don't think we need the last comment. There's no good reason for someone to use a different time format for hrSystemDate.

Reply:
Okay.
I will delete the comment.